src/pipelinedefinition.cpp

```diff
@@ -161,12 +162,20 @@ Status PipelineDefinition::create(std::unique_ptr<Pipeline>& pipeline,
             break;
         }
         case NodeKind::DL:
-            nodes.insert(std::make_pair(info.nodeName, std::move(std::make_unique<DLNode>(info.nodeName,
+            nodes.insert(std::make_pair(info.nodeName, std::move(std::make_unique<DLNode>(
```

Comment:
Minor:
I think we should race std::unorded_map::insert with replace & then we could remove this std::move as well.

Reply:
Removed std::move from DL & CUSTOM node